flexmeasures/data/models/charts/belief_charts.py

```diff
@@ -112,6 +112,15 @@ def chart_for_multiple_sensors(
             ]
         }
 
+    # Set up field definition for sensor descriptions
+    sensor_field_definition = FIELD_DEFINITIONS["sensor_description"]
```

Comment:
I'm not sure if the dictionary `sensor_field_definition` keeps the reference to the original data. In such case, we could encounter interference when multiple users are accessing.

Reply:
Great catch!

```
specs = dict(foo=dict(bar=1))
field = specs["foo"]
field["bar"] = 0 
print(specs)  # {'foo': {'bar': 0}}
```

Should be:

```
specs = dict(foo=dict(bar=1))
field = specs["foo"].copy()
field["bar"] = 0 
print(specs)  # {'foo': {'bar': 1}}
```